src/AttributesBlockParser.php

```diff
@@ -26,7 +26,7 @@ public function parse(ContextInterface $context, Cursor $cursor)
             return false;
         }
 
-        if (null !== $cursor->getFirstNonSpaceCharacter()) {
+        if (null !== $cursor->getNextNonSpaceCharacter()) {
```

Comment:
Looks like this is broken on 0.14?

Reply:
Updated to remove 0.14 in the dependency tree